aspnetcore/security/enforcing-ssl.md

```diff
@@ -1003,6 +915,8 @@ For chromium browsers on Linux:
 
 ### Trust the certificate with Firefox on Linux
 
+In some instances the Firefox browser uses its own certificate store, and therefore doesn't trust the [IIS Express](/iis/extensions/introduction-to-iis-express/iis-express-overview) or [Kestrel](xref:fundamentals/servers/kestrel) developer certificates and requires the following instructions. 
+
```

Comment:
@amcasey and @tdykstra  I think  in the FF section we should add something like the following:

In some instances theses instructions don't work with Firefox on Linux. See [Trust the HTTPS certificate with Firefox](https://learn.microsoft.com/en-us/aspnet/core/security/enforcing-ssl?view=aspnetcore-5.0&tabs=visual-studio%2Clinux-ubuntu#trust-the-https-certificate-with-firefox-to-prevent-sec_error_inadequate_key_usage-error-1] for alternative instructions. The link goes back to .NET 5 and has all the previous instructions.

Reply:
AFAIK, the old instructions apply to old version of Firefox, rather than to old versions of .NET.  If you're on a new enough .NET that `--trust` works, no Firefox-specific steps should be required.  Otherwise, the instructions should be as for Chromium, but with `"C,,"` and different locations.

Or maybe I've misunderstood your suggestion?  I'm not entirely sure where in the document this text occurs.

Reply:
> AFAIK, the old instructions apply to old version of Firefox, rather than to old versions of .NET. If you're on a new enough .NET that `--trust` works, no Firefox-specific steps should be required. Otherwise, the instructions should be as for Chromium, but with `"C,,"` and different locations.
> 
> Or maybe I've misunderstood your suggestion? I'm not entirely sure where in the document this text occurs.

Great, we'll replace that with older versions of FF